internal/host/installcmd_test.go

```diff
@@ -160,7 +160,7 @@ func validateInstallCommand(reply *models.Step, role models.HostRole, clusterId
 			"--name assisted-installer quay.io/ocpmetal/assisted-installer:latest --role %s " +
 			"--cluster-id %s " +
 			"--boot-device /dev/sdb --host-id %s --openshift-version 4.5 " +
-			"--controller-image %s --url %s --host-name %s"
+			"--controller-image %s --url %s --insecure=false --host-name %s"
```

Comment:
I think we should have a case where SkipCertificateVerification == True

Reply:
This unit-test tests something else, so the default is here. For the right uni-test see https://github.com/empovit/assisted-service/blob/d32b122da86aaebb8c9a54ce13f792d73d8767c2/internal/bminventory/inventory_test.go#L297

Reply:
this UT does not testa the case when the configuration has True value for SkipCertificateVerification